test/e2e/network/service.go

```diff
@@ -518,6 +518,110 @@ func pokeUDP(host string, port int, request string, params *UDPPokeParams) UDPPo
 	return ret
 }
 
+// continuousEcho() uses the same connection for multiple requests, made to run as a goroutine so that
+// manipulations can be made to the service and backend pods while a connection is ongoing
+func continuousEcho(host string, port int, timeout time.Duration, maxAttempts int, errorChannel chan error) {
+	hostPort := net.JoinHostPort(host, strconv.Itoa(port))
+	url := fmt.Sprintf("udp://%s", hostPort)
+
+	ret := UDPPokeResult{}
+
+	// Sanity check inputs, because it has happened.  These are the only things
```

Comment:
nit , should we do the sanity check at the beginning, before initializing the variables?

```
hostPort := net.JoinHostPort(host, strconv.Itoa(port))
url := fmt.Sprintf("udp://%s", hostPort)
```

Reply:
added the sanity check above the variable initialization